spi_nand_flash/src/spi_nand_oper.h

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
This header is missing C++ guards. (Won't comment on other headers for this issue, please check yourself.)